src/artofillusion/tapDesigner/TapProcedure.java

```diff
@@ -1082,8 +1078,8 @@ public void importTextures( BFrame parentFrame )
      */
     public void manageTextures( BFrame parentFrame )
     {
-        theScene.showTexturesDialog( parentFrame );
-        setModified( true );
+        //theScene.showTexturesDialog( parentFrame );
```

Comment:
These can probably be similar to the ones in the Texture and Material manager dialogs.